### ISSUE TITLE
Git Clone is approx 30GB++

### ISSUE THREAD

**Issue by Rcreators** [Unity-Technologies/FPSSample#23]
Hello,

Is it me or other people also getting clone more then 30 GB.

Yesterday i put the git to clone with git lfs. I added this commands in sequence

1) git lfs install - which initialized lfs 

2) git clone https://github.com/Unity-Technologies/FPSSample.git

Above command started the cloning of project on my local desktop. So i went for sleep, after i woke up, i saw a error that my disk have no space and so clone stopped. So look the properties of folder and its more than 30 GB.

http://prntscr.com/ldv12x

Can anybody explain this ? or do i made any mistake in above commands ?

**Comment by Rcreators** [comment]
Thanks. Its so confusing cause they written in read me that its only 18 GB. Now how i can resume clone ? any command for it. Cause its not completed. I only have 30 GB while as you described its 34 GB around.

**Comment by Fijit** [comment]
Just to help clarify for anyone still confused
TLDR; You will need 45GB+ to get going.

The git repository is ~18GB
Once the files are checked out into /assets etc. they will be a copy of the current state of each file from the .git folder thereby more or less doubling the size on disk.
That's ~36GB
Another ~4GB to build /library = 40GB
AutoBuilds will probably add another 4.5GB = 45.5GB

So call it 50GB so you have room to play.

Also note: it was mentioned in the setup instructions...but not very clearly as you mentioned.

`The project size is about 18GB (size of Assets folder). Your cloned repository will be almost double of that due to git state. If it is much smaller, you most likely did not have LFS when you cloned.`

Hope that helps :)

**Comment by Rcreators** [comment]
Thank fijit for details. Can anyone let me know if run the command again, will it resume clone process or start from first byte ? Cause can't download 30gb again. If there is any command which can just download remaining files.

**Comment by NNNNNNNNNNNNNNNNNNNNN** [comment]
Use the right version..
